tests/common_test.h

```diff
@@ -0,0 +1,308 @@
+/*common_test.h */
+
```

Comment:
this needs to be refactored to work with rpm files.  Please only select the functions that are needed for the tests.